opencog/atoms/base/ClassServer.h

```diff
@@ -210,6 +253,21 @@ class ClassServer
 
 ClassServer& classserver();
 
+#define DEFINE_LINK_FACTORY(CNAME,CTYPE)                          \
+                                                                  \
```

Comment:
@linas searched the repository and build directory for the use of this macro in the creation of atoms such as ConceptNode, couldn't find any. What is the magic here?